SparkPost/matchbox
# UX-707 Take Percy snapshots of Modal and Legacy Modal

### What Changed

- Takes a Percy snapshot of Modal and Legacy Modal

### How To Test or Verify

- Verify the percy build
- - https://percy.io/efff6832/matchbox/builds/13125839/changed/741361213?browser=edge&browser_ids=18%2C19%2C20%2C21&subcategories=approved&viewLayout=side-by-side&viewMode=new&width=1280&widths=375%2C1280

### PR Checklist
